CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 0.2.1 (September 22, 2015) ##
 
+## Bug fix
+* Fixes regression in omniauth-oauth2 v1.4.0 by [intridea/omniauth-oauth2@85fdbe1](https://github.com/intridea/omniauth-oauth2/commit/85fdbe1)
+
+## 0.2.1 (September 22, 2015) ##
```

Comment:
looks like you copy-pasted version as well here :) 
Can you just put that README addition to the `## Unreleased` section?